g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties

```diff
@@ -32,6 +32,7 @@ server.trace.enable=false
 #collection source server
 source.cache.flag = false
 source.cache.server.url = https://localhost:8088
+source.request.max-size = 10485760
```

Comment:
By default, it doesn't need to set the size, comment out this setting?

Reply:
done

Reply:
done